modmesh/app/euler1d.py

```diff
@@ -61,73 +66,315 @@ def load_app():
-        self.ax = self.canvas.figure.subplots()
-        self.ax.set_ylim(-0.2, 1.2)
-        self.ax.grid()
+class PlotManager(QWidget):
+    def __init__(self, shocktube):
+        super().__init__()
+        self.setWindowTitle("Euler 1D")
+        self.setMinimumSize(500, 350)
+        self.setAcceptDrops(True)
+
+        self.scrollArea = QScrollArea()
+        self.scrollArea.setWidgetResizable(True)
+
+        self.layout = QVBoxLayout()
+        self.setLayout(self.layout)
+
+        self.graphContainer = QWidget()
+        self.use_gridLayout = True
+        self.graphLayout = None
+        self.shocktube = shocktube
+
+        self.scrollArea.setWidget(self.graphContainer)
+        self.layout.addWidget(self.scrollArea)
+
+        self.button_layout = QHBoxLayout()
+
+        self.save_button = QPushButton()
+        self.save_button.setText("Save")
+
+        self.swtich_button = QPushButton()
+        self.swtich_button.setText("Layout swtich")
+
+        self.button_layout.addWidget(self.save_button)
+        self.button_layout.addWidget(self.swtich_button)
+
+        self.save_button.clicked.connect(self.save_all)
+        self.swtich_button.clicked.connect(self.switch_layout)
+
+        self.layout.addLayout(self.button_layout)
 
         self.density = QuantityLine(name="density",
                                     unit=r"$\mathrm{kg}/\mathrm{m}^3$")
         self.velocity = QuantityLine(name="velocity",
                                      unit=r"$\mathrm{m}/\mathrm{s}$")
         self.pressure = QuantityLine(name="pressure", unit=r"$\mathrm{Pa}$")
+        self.temperature = QuantityLine(name="temperature",
+                                        unit=r"$\mathrm{K}$")
+        self.int_energy = QuantityLine(name="int. energy",
+                                            unit=r"$\mathrm{J}/\mathrm{kg}$")
+        self.entropy = QuantityLine(name="entropy",
+                                    unit=r"$\mathrm{J}/\mathrm{K}$")
+
+    def save_all(self):
+        fig = QPixmap(self.graphContainer.size())
+        self.graphContainer.render(fig)
+
+        options = QFileDialog.Options()
+        options |= QFileDialog.DontUseNativeDialog
+        fileName, _ = QFileDialog.getSaveFileName(None, "Save file", "",
+                                                  "All Files (*)",
+                                                  options=options)
+
+        if fileName != "":
+            fig.save(fileName, "JPG", 100)
+
+    def switch_layout(self):
+        while self.graphLayout is not None and self.graphLayout.count():
+            item = self.graphLayout.takeAt(0)
+            if isinstance(item, QLayout):
+                self.delete_layout(item)
+            else:
+                item.widget().setParent(None)
+                item.widget().deleteLater()
+
+        self.graphLayout.deleteLater()
+        self.graphContainer.update()
+
+        self.use_gridLayout = not self.use_gridLayout
```

Comment:
I think we should not use both `snake_camelCase` style in one name.  Could you please change it to either `snake_case` or `camelCase`?

Reply:
ditto.

Reply:
I am not seeing it fixed in the review.

Reply:
I fixed it in https://github.com/solvcon/modmesh/pull/210/commits/b9ea25486ed643d5ee0974dd8d17c1f22f4285e8